deploy/application/java/java-war.md

```diff
@@ -110,49 +149,57 @@ Here's what your configuration file can look like:
 ```
 
 <table class="table table-bordered table-striped">
-<thead>
-<tr>
-<th>Usage</th>
-<th>Field</th>
-<th>Description</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td><span class="label label-danger">Required</span></td>
-<td>**container**</td>
-<td>Name of the container to use. Should contain one of the values inside parentheses in the [containers table](#available-containers) (in uppercase and all).</td>
-</tr>
-<tr>
-<td><span class="label label-danger">Required</span></td>
-<td>**file**</td>
-<td>Should contain the path of the war/ear file relative to your application root.</td>
-</tr>
-<tr>
-<td><span class="label label-default">Optional</span></td>
-<td>**context**</td>
-<td>
-<ul>
-<li>Must start with a slash (/), can be "/".</li>
-<li>Defines the base path you want your app to be under. If your app has a /foobar endpoint, it will be available under the `/{my-context}/foobar` path.</li>
-<li>Not needed for an `ear` file.</li>
-<li>The default value for a war is the name of the war without the extensions: helloworld-1.0.war will be deployed under the `/helloworld-1.0` context.</li>
-</ul>
-</td>
-</tr>
-<tr>
-<td><span class="label label-default">Optional</span></td>
-<td>**checkMe**</td>
-<td>
-<ul>
-<li>This field is recommended</li>
-<li>A path to GET in order to test if the application is really running.</li>
-<li>By default we will consider that the application is up if the container is up.</li>
-<li>With this option, we will try to `GET /{checkMe}` for each one of your wars and consider the app down until every single checkMe path that replies a 200.</li>
-</ul>
-</td>
-</tr>
-</tbody>
+   <thead>
+   <tr>
+   <th>Usage</th>
+   <th>Field</th>
+   <th>Description</th>
+   </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>
+            <span class="label label-danger">Required</span>
+         </td>
+         <td>**container**</td>
```

Comment:
The `**` are rendered here too but I'm not sure we want those words to be strong anyway. Maybe we can just drop them, I don't know. If you apply something here, you can also do it for the rest of the fields